rubocop.yml

```diff
@@ -35,7 +35,7 @@ Style/FrozenStringLiteralComment:
     - 'db/migrate/*.rb'
 
 Metrics/MethodLength:
-  Max: 15
```

Comment:
Disagree, 15 sounds like a reasonable numbers of lines for one method. Any examples?

Reply:
Take a look at resolve methods in mutations. You will see giant hashes as a result of action. This hashes is unreadable if we trying to fit them in 15 lines